build_image.py

```diff
@@ -224,8 +253,20 @@ def create_dockerfile_finish(config: OfrakImageConfig) -> str:
             "\\n",
         ]
     )
-    dockerfile_finish_parts.append(f'RUN printf "{develop_makefile}" >> Makefile\n')
-    dockerfile_finish_parts.append("RUN make $INSTALL_TARGET\n\n")
+    dockerfile_finish_parts += [
+        f'RUN printf "{develop_makefile}" >> Makefile\n\n',
+        ### The --network="none" is very helpful in catching all kinds of issues,
+        ### and should probably be enabled eventually.
+        ### Unfortunately it cases problems when a package has a pyproject.toml -
+        ### see https://github.com/pypa/pip/issues/6482#issuecomment-1414208986
+        ### A workaround is to use `pip install --no-build-isolation` or
+        ### `pip install --no-use-pep517` but that starts getting messy.
+        ### Leaving it as commented code here to make it easier for people
+        ### to temporarily enable it.
+        #'# We use --network="none" to ensure all dependencies were installed in base.Dockerfile\n',
+        #'RUN --network="none" make $INSTALL_TARGET\n\n',
```

Comment:
```suggestion
```
This isn't a comment explaining what the code does, but an aside about future improvements. Can you memorialize it as a comment on the PR or, if you think it merits one, an issue on the GitHub tracker?

Reply:
I've made `--network="none"` this a configurable option (off by default).